### DISCUSSION TITLE
How to install rabbitmq NOT as a Windows Service

### DISCUSSION THREAD

**Discussion by alan-kozlay** [rabbitmq/rabbitmq-server#11880] [Questions] [answered]
After install is complete, I need to be able to start rabbitmq with rabbitmq-server.bat instead of allowing it to start as a Windows Service.  But the installer defaults to enabling it as windows service.  For important reasons, I can't simply follow the install with "rabbitmq-service.bat disable".  Is there a switch/param I can provide on the command-line when kicking off the rabbitmq installer "RabbitMQ.exe /S" which will prevent it from enabling it as a Windows Service?

**Comment by lukebakken** [comment, marked as answer]
https://github.com/rabbitmq/rabbitmq-packaging/blob/main/windows-exe/rabbitmq_nsi.in#L331-L340

**Reply by alan-kozlay** [reply]
Great!  Thank you.
